src/test/java/ui/UserAccountResult.java

```diff
@@ -0,0 +1,44 @@
+package ui;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserAccountResult {
+    @NonNull
+    private String firstName;
+    @NonNull
+    private String lastName;
+    @NonNull
+    private String email;
+    @NonNull
+    private String day;
+    @NonNull
+    private String month;
+    @NonNull
+    private String year;
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
wow... Why not `@EqualsAndHashCode` from lombok? https://projectlombok.org/features/EqualsAndHashCode

Reply:
I think I forgot (

I added annotation.